docs/docs/30-how-to-guides/14-working-with-stages.md

```diff
@@ -0,0 +1,548 @@
+   The new `Stage` will be added to the pipeline, connected to other
+   `Stage`s based on your configuration:
+
+   ![create-stage](../../static/img/create-stage-3.png)
+</TabItem>
+
+<TabItem value="cli" label="Using the Kargo CLI">
+
+1. Define the `Stage` in a YAML file, for example:
+
+```yaml
+apiVersion: kargo.akuity.io/v1alpha1
+kind: Stage
+metadata:
+  name: <stage-name>
+  namespace: <project-name>
+spec:
+  ### Add your Stage specifications here
+```
+
+2. Save the file and run:
+
+```shell
+kargo create -f <stage-filename>
+```
+
+3. Verify the creation by listing the `Stage`:
+
+```shell
+kargo get stage <stage-name> --project <project-name>
+```
+
+</TabItem>
+</Tabs>
+
+### Deleting a Stage
+
+<Tabs groupId="delete-stage">
+<TabItem value="ui" label="Using the Kargo Dashboard" default>
+
+1. Select the `Stage` you want to remove.
+2. Click <Hlt>Delete</Hlt> in the top-right corner of the pop-up window:
+
+   ![delete-stage](../../static/img/delete-stage.png)
+</TabItem>
+
+<TabItem value="cli" label="Using the Kargo CLI">
+To delete a `Stage` using the CLI, run:
+
+```shell
+kargo delete stage --project <project-name> <stage-name>
+```
+
+</TabItem>
+</Tabs>
+
+### Refreshing a Stage
+
+Refreshing a `Stage` triggers a reconciliation process that
+performs essential tasks, including checking for completed 
+Promotions, running health checks, verifying the current 
+Freight, and determining if any pending Promotions can proceed. 
+This ensures that the `Stage` reflects the latest state and operational readiness.
+
+<Tabs groupId="refresh-stage">
+<TabItem value="ui" label="Using the Kargo Dashboard" default>
+1. Select the `Stage` you want to refresh.
+2. Click <Hlt>Refresh</Hlt> in the top-right corner of the pop-up window:
+
+   ![refresh-stage](../../static/img/refresh-stage.png)
+</TabItem>
+
+<TabItem value="cli" label="Using the Kargo CLI">
+To refresh a `Stage`, run:
+
+```shell
+kargo refresh stage --project=<project-name> <stage-name>
+```
+
+</TabItem>
+</Tabs>
+
+### Reverifying a Stage's Current Freight
+
+Reverification ensures that the `Freight` associated with a `Stage` is functioning as expected. This can include rerunning or aborting verification.
+
+<Tabs groupId="verify-stage">
+<TabItem value="ui" label="Using the Kargo Dashboard" default>
+1. Select the `Stage` you want to reverify and click <Hlt>Reverify</Hlt> at the top of the menu:
+
+![verify-stage](../../static/img/reverify-freight.png)
+
+2. To confirm the `Stage(s)` where the `Freight` has been verified, return
+to the <Hlt>Freight Timeline</Hlt> and select the `Freight`.
+Verified `Stage` names will appear under <Hlt>VERIFIED IN</Hlt>:
+
+![verify-stage](../../static/img/verified-in.png)
+</TabItem>
+
+<TabItem value="cli" label="Using the Kargo CLI">
+1. To rerun verification using the CLI run:
+
+```shell
+kargo verify stage --project=<project-name> <stage-name>
+```
+
+2. To stop an ongoing verification process, use:
```

Comment:
This is awkward. It's being presented as:

Step 1: Start verification
Step 2: Abort it

Those feel like two different things, rather than elements of the same sequence.

This also differs from the UI-centric instructions above, which didn't talk about aborting a verification, but went from starting one to showing how you can see which Stages a piece of Freight has been verified in.

We should try to keep things consistent and have the UI and CLI tabs cover the same things in the same order.